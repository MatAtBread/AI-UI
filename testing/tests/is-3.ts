/// <reference path="../test.env.d.ts"/>

for await (const x of Test.count()) {
    console.log("The expected result is", x)
}

export { };