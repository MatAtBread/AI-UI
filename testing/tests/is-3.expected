[
  ["\"The expected result is\"","0"],
  ["\"The expected result is\"","1"],
  ["\"The expected result is\"","2"],
  ["\"The expected result is\"","3"],
  ["\"The expected result is\"","4"],
  ["\"The expected result is\"","5"],
  ["\"The expected result is\"","6"],
  ["\"The expected result is\"","7"],
  ["\"The expected result is\"","8"],
  ["\"The expected result is\"","9"]
]